Orange/preprocess/score.py

```diff
@@ -207,7 +207,7 @@ def _gini(D):
     """Gini index of class-distribution matrix"""
     P = D / np.sum(D, axis=0)
     return sum((np.ones(1 if len(D.shape) == 1 else D.shape[1]) - np.sum(np.square(P), axis=0))
-               * 0.5 * np.sum(D, axis=0) / np.sum(D))
+               * np.sum(D, axis=0) / np.sum(D))
```

Comment:
This sums up [the proposed equation](https://en.wikipedia.org/wiki/Decision_tree_learning#Gini_impurity) somewhat more clearly:
```py
return 1 - (P*P).sum()
```